airflow/providers/pinecone/CHANGELOG.rst

```diff
@@ -20,6 +20,29 @@
 Changelog
 ---------
 
+2.0.0
+.....
+
+Breaking changes
+~~~~~~~~~~~~~~~~
+
+.. note::
+   This release of provider has breaking changes from previous versions. Changes are based on
+   the migration guide from pinecone - <https://canyon-quilt-082.notion.site/Pinecone-Python-SDK-v3-0-0-Migration-Guide-056d3897d7634bf7be399676a4757c7b>
```

Comment:
Could you also add what is changed and what actions/steps required to do by end users to migrate to new version

Some examples:

- https://github.com/apache/airflow/blob/main/airflow/providers/apache/hive/CHANGELOG.rst#breaking-changes
- https://github.com/apache/airflow/blob/main/airflow/providers/amazon/CHANGELOG.rst#800
- https://github.com/apache/airflow/blob/main/airflow/providers/slack/CHANGELOG.rst#800
- https://github.com/apache/airflow/blob/main/airflow/providers/google/CHANGELOG.rst#breaking-changes

Reply:
Thank you. I'll go through them and update